sdk/monitor/azure-monitor-opentelemetry-exporter/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 1.0.0-beta.4 (Unreleased)
-
+- Updated versions of `opentelemetry-api` and `opentelemetry-sdk` to `0.16.0` version.
```

Comment:
0.17.1

Reply:
Add a link to OTel release notes as well.